beginner/cogs/buddy.py

```diff
@@ -142,6 +144,111 @@ async def archive_buddy_chat(self, ctx: nextcord.ext.commands.Context):
         await ctx.channel.edit(name=f"{ctx.channel.name}-archive")
         await ctx.channel.send("🗂 This channel has been archived")
 
+    @Cog.slash_command(
```

Comment:
Can the slash command be disabled until such a time that the Select's will work inside the Modal?

Also it is possible to restrict it to a certain channel since the response's all ideally go into the same channel?

Reply:
> Can the slash command be disabled until such a time that the Select's will work inside the Modal?
Yes! You can change the command permissions to restrict usage from certain roles in the "Integrations" settings.

> Also it is possible to restrict it to a certain channel since the response's all ideally go into the same channel?
As per your request from chat, I implemented a way to send the response to a specific channel while also leaving an ephemeral message mentioning that channel.